hack/make/test-integration-flaky

```diff
@@ -24,6 +24,10 @@ echo "Running stress test for them."
     # This will make a total of 25 runs for each test in TESTARRAY.
     export TEST_REPEAT=5
     local testcount=5
+    # Test-daemon is not started yet, it's not possible to determine os/arch by
+    # calling hack/make/.detect-daemon-osarch and to determine architecture specific TIMEOUT.
+    # Use default value 5m if TIMEOUT is not specified.
+    : ${TIMEOUT:=5m}
```

Comment:
This one is tricky indeed 🤔 

Not sure what's best; should we make `set_platform_timeout()` look for `$TEST_REPEAT`, and if it's set perform the calculation inside `set_platform_timeout` ?

@tianon perhaps you have ideas?

Reply:
Yeah, I agree that looking at `TEST_REPEAT` at that lower level instead probably makes sense here to factor for those "slower" architectures (and from the looks of it, we can ditch the `testcount` variable entirely since it's just a local copy of `TEST_REPEAT` that isn't exported?)

I did some grepping to make sure, and this is the only place we use `TEST_REPEAT` other than `hack/make/.integration-test-helpers` where the actual repeating happens, so we should be safe to key off of that directly in the check.

Reply:
Agree.
I put the handling of `TEST_REPEAT` into `set_platform_timeout()` as well. See my latest code.